### PR TITLE
Added reporting to consensus context

### PR DESCRIPTION
**Pull request by IdoZilberberg**
Also replaced Printf()'s with reporting.Info().

### CODE REVIEW

File: services/consensuscontext/service.go [hunk 1]
```diff
@@ -38,8 +41,7 @@ func (s *service) RequestNewTransactionsBlock(input *services.RequestNewTransact
 		return nil, err
 	}
 
-	// TODO: add reporting mechanism to this service and convert this print to it
-	fmt.Printf("created Transactions block num-transactions=%d block=%s\n", len(txBlock.SignedTransactions), txBlock.String())
+	s.reporting.Info("created Transactions block", instrumentation.Int("num-transactions", len(txBlock.SignedTransactions)), instrumentation.String("block", txBlock.String()))
```
Comment (netoneko) [inline services/consensuscontext/service.go]: Why `txBlock.String()`? We have `instrumentation.Stringable`.

File: services/consensuscontext/service.go [hunk 2]
```diff
@@ -52,8 +54,7 @@ func (s *service) RequestNewResultsBlock(input *services.RequestNewResultsBlockI
 		return nil, err
 	}
 
-	// TODO: add reporting mechanism to this service and convert this print to it
-	fmt.Printf("created Results block: %s\n", rxBlock.String())
+	s.reporting.Info("created Results block", instrumentation.String("results-block", rxBlock.String()))
```
Comment (netoneko) [inline services/consensuscontext/service.go]: Same here.

File: services/consensuscontext/service.go (hunk 1 above)
Comment (IdoZilberberg) [reply services/consensuscontext/service.go]: Thanks, fixed.

Comment (IdoZilberberg) [reply services/consensuscontext/service.go]: Thanks, fixed.

File: services/consensuscontext/service.go [hunk 3]
```diff
@@ -38,8 +41,7 @@ func (s *service) RequestNewTransactionsBlock(input *services.RequestNewTransact
 		return nil, err
 	}
 
-	// TODO: add reporting mechanism to this service and convert this print to it
-	fmt.Printf("created Transactions block num-transactions=%d block=%s\n", len(txBlock.SignedTransactions), txBlock.String())
+	s.reporting.Info("created Transactions block", instrumentation.Int("num-transactions", len(txBlock.SignedTransactions)), instrumentation.Stringable("block", txBlock))
```
Comment (netoneko) [inline services/consensuscontext/service.go]: I suggest you use `transactions-block` instead of `block`.

**Pull request merged**